packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -121,8 +131,40 @@ export const connectEpic: Epic<AnyAction, AnyAction, any, any> = (action$, state
         }),
       )
 
+      const messages$ = action$.pipe(
```

Comment:
I would put this functionality into the `multiplex` pipeline above, since its tightly bound to that anyway when the `messageReceived` function is called